tests/lang/s01_basics/s06_data_types/t7_strings.nim

```diff
@@ -10,7 +10,7 @@ string type related tests, covering key properties around about:
 target: "c cpp js"
 """
 
-block:
+block stringsValid_1:
```

Comment:
Don't number valid string construction examples, name them. Like `string_valid_from_empty_string`

Reply:
Ye I just don't know what to name them, thought I would put these in as placeholders so if the block name is referenced in any kind of output, then it would be easier to locate

Reply:
Should I revert them to blank?

Reply:
You should turn them into `strings_valid_from_empty_string`, `string_vm_and_runtime_handling`, `string_concatenation`, `string_unicode_handling` and `string_copy_on_assignment`